services/hierarchy.js

```diff
@@ -9,15 +9,18 @@ const hierarchySerive = {
       return { message: "error", data: error.message };
     }
   },
-
-  get: async (id) => {
-    try {
-      const savedData = await hierarchyModel.findByIdAndDelete(id);
-      if (savedData) {
-        return { message: "success", data: savedData };
-      }
-    } catch (error) {
-      return { message: "error", data: error.message };
+  get: async (id)=>{
+    try{
+        const data = await hierarchyModel.findById(id);
+        if(data){
+        return { message: "success", data };

```

Comment:
Do not send message from service layer. just send data. handle message in controller.